qiskit/pulse/parameter_manager.py

```diff
@@ -146,7 +146,6 @@ def visit_Schedule(self, node: Schedule):
         """Visit ``Schedule``. Recursively visit schedule children and overwrite."""
         # accessing to private member
         node._Schedule__children = [(t0, self.visit(sched)) for t0, sched in node.instructions]
```

Comment:
Oops I missed this last time -- can we do the same trick here? Where we ask Schedule to do this itself, but call to the ParameterManger (or Visitor) to do this `visit` work